invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/errors/FormFeedback.js

```diff
@@ -266,8 +266,20 @@ class DisconnectedFormFeedback extends Component {
       </Message.Item>
     ));
 
-    // errors not related to validation, following a different format {status:.., message:..}
-    const backendErrorMessage = errors.message;
+    // errors not related to validation, following a different format {status:..., message:..., errors: {field: ..., messages: [...]}}
+    let backendErrorMessage = errors.message;
+
+    // add extra backend error messages related to fields if present
+    if (backendErrorMessage && errors.errors) {
+      backendErrorMessage +=
+        " " +
```

Comment:
minor: should we do a bullet-list instead (similar to the usual form validation errors), since it's an array?